multiDOF[] points
int64 traj_id
Header header